### DISCUSSION TITLE
Giorni festivi, problemi con template data

### DISCUSSION THREAD

**Discussion by bozzettaccio** [indomus/forum#489] [HUB: Home Assistant]
Ciao, ho implementato il binary_sensor workday per automatizzare i giorni in cui mettere la sveglia. Oltre ai giorni festivi nazionali preimpostati con "country: IT" volevo aggiungere il giorno del patrono della città ma la funzione add_holdays prevede l'inserimento del giorno con l'anno nella forma che segue:
```
  - '2019-06-29'
```
. Per renderlo variabile ogni anno senza dover ogni volta cambiare il binary sensore, l'avevo impostato secondo variabile nel modo che ho commentato di seguito ma, ovviamente, non funziona nonostante nel test di template esce esattamente nella forma indicata sopra.
```
- platform: workday
  name: domani_lavorativo
  country: IT
  workdays: [mon, tue, wed, thu, fri]
  excludes: [sat, sun, holiday]
  #add_holidays:
  #  - '{{ now().year }}-06-29'
  days_offset: 1
```
sapete in che modo debba scriverlo per farlo funzionare?
Grazie mille!!!

**Comment by 50494554524F** [comment]
mettine per 10 anni e via...

**Comment by maxcanna** [comment]
- ```
    workdays: [mon, tue, wed, thu, fri]
    excludes: [sat, sun, holiday]
  ```
  sono inutili in quanto già valori di default

- ```
  add_holidays:
  ```
  non supporta l'uso di un template come puoi vedere in doc

- imposta
  ```
  province: RM
  ```
  ed avrai il 29/06 sempre festivo
